src/commands/install.yml

```diff
@@ -1,22 +1,26 @@
 description: >
-  Install MATLAB on a Linux machine executor. Currently, this command is available only for public
-  projects and does not include transformation products, such as MATLAB Coder and MATLAB Compiler.
+  Install a specific version of MATLAB. If you use this command to install transformation products, such
+  as MATLAB Coder and MATLAB Compiler, the command does not automatically license such products for you.
 
 parameters:
   products:
     description: >
-      List of MathWorks products to install.
+      Products to install in addition to MATLAB, specified as a list of product names separated by
+      spaces. You can specify this parameter to install most MathWorks products and support packages. The
+      command uses MATLAB Package Manager (`mpm`) for installing products. For a list of supported
+      products and their correctly formatted names, see
+      https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md#product-installation-options.
     type: string
-    default: 'MATLAB'
+    default: ''
   release:
     description: >
-      MATLAB release to install. You can specify R2020a or a later release. By default, the command
+      MATLAB release to install. You can specify R2021a or a later release. By default, the command
```

Comment:
MATLAB release to install. You can specify R2021a or a later release. By default, the value of `release` is `latest`. If you do not specify `release`, the command installs the latest release of MATLAB.